### PR TITLE
Register Subdomain: snek

### PR DESCRIPTION
**Pull request by sn3kdev**
Added the domain: snek

### CODE REVIEW

File: domains/snek.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "sn3kdev",
+        "email": "snektg@gmail.com"
+      },
+      "record": {
+        "URL": ""odnako_zmeyka.t.me""
```
Comment (WilliamDavidHarrison) [inline domains/snek.json:8]: Please remove the extra quotations and add `https://` infront of the link

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/snek.json (hunk 1 above)
Comment (sn3kdev) [reply domains/snek.json:8]: Had committed

Comment (sn3kdev) [reply domains/snek.json:8]: https://github.dev/sn3kdev/register/tree/snek-https

Comment (WilliamDavidHarrison) [inline domains/snek.json:8]:
```suggestion
        "URL": "https://odnako_zmeyka.t.me"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

**Review by TheBotlyNoob — APPROVED** [review]

**Pull request merged**